src/Scene/Scene.js

```diff
@@ -85,6 +86,21 @@ Scene.prototype.getPickPosition = function(mouse) {
     return this.gfxEngine.getPickingPositionFromDepth(mouse);
 };
 
+Scene.prototype.getPickFeature = function(Position, Layer){
```

Comment:
There is already a picking mechanism implemented, start here: https://github.com/iTowns/itowns2/blob/master/src/Renderer/c3DEngine.js#L410

I implemented building picking in another branch if it's any help : https://github.com/iTowns/itowns2/commit/a075c1b03607f5686651b7d6498f57980e331ae5